src/script/conversation/ConversationRepository.ts

```diff
@@ -1852,10 +1875,6 @@ export class ConversationRepository {
 
       conversation.connection(connectionEntity);
 
-      if (connectionEntity.isConnected()) {
-        conversation.type(CONVERSATION_TYPE.ONE_TO_ONE);
-      }
```

Comment:
This part was moved to `initProteus1to1Conversation` method - no need to run this on mls 1:1 - it's always of type `ONE_TO_ONE`.